spec/tictactoe_spec.rb

```diff
@@ -0,0 +1,243 @@
+# frozen_string_literal: true
+
+require 'io/console'
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+
+describe 'board' do
+  mplayer1 = Player.new('X')
+  mplayer2 = Player.new('O')
+  my_board = Board.new(mplayer1, mplayer2)
+
+  it 'testing 1st row for player1' do
+    my_board.reset
+
+    my_board.mark(0)
```

Comment:
- [ ] Specs for each method should be wrapped in one describe block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples .